### DISCUSSION TITLE
new tab button

### DISCUSSION THREAD

**Discussion by guilleya** [onemen/TabMixPlus#344] [General]
I notice in my FF that when I set the tabs in row in a maximun number of rows and exceed that number of rows, .the new tab button dissapared.... has anyone else experienced the same... thanks

**Comment by pyrates999** [comment]
First please try the dev build here: https://github.com/onemen/TabMixPlus/releases

And enable the option: `Tab width fills rows on overflow` under Display -> Tab.

Then please specify your firefox version.

And upload your tab mix plus preferences if this doesn't fix it.

**Comment by onemen** [comment]
@guilleya

What is your Firefox version and platform,

when your tabs are in overflow state in multi-row  mode, the button should be on the right side.

![image](https://github.com/user-attachments/assets/4436343f-8bb7-4546-995b-ac8a7b2e9ba6)

**Comment by guilleya** [comment]
Thank you for the advice... I found the problem... it was that I have chrome folder with other files ....
